scripts/update-versions.mjs

```
@@ -0,0 +1,66 @@
+// run from the root of the repository: node scripts/update-versions.mjs 0.12.0.alpha-1
+const newVersion = process.argv[2];
+
+console.info('Updating version in various files, version:', newVersion);
+updateVersionInRootPackageLockJsonFile(newVersion);
+updateVersionInCorePackageJsonFile(newVersion);
+updateVersionInSourceFile(newVersion);
+console.info('Files have been updated');
+
+function updateVersionInRootPackageLockJsonFile(newVersion) {
+  const path = 'package-lock.json';
+  console.info('Updating', path);
```

ESLint

[error] Unexpected console statement.

(no-console)